lwr_hw/src/lwr_hw_fri_node.cpp

```diff
@@ -106,14 +107,17 @@ int main( int argc, char** argv )
   ros::Time last(ts.tv_sec, ts.tv_nsec), now(ts.tv_sec, ts.tv_nsec);
   ros::Duration period(1.0);
 
+  float sampling_time = lwr_robot.getSampleTime();
+  ROS_INFO("Sampling time on robot: %f", sampling_time);
+
   //the controller manager
   controller_manager::ControllerManager manager(&lwr_robot, lwr_nh);
 
   // run as fast as possible
   while( !g_quit )
   {
     // get the time / period
-    if (!clock_gettime(CLOCK_REALTIME, &ts))
+    if (!clock_gettime(CLOCK_MONOTONIC, &ts))
```

Comment:
I see...  just reading all discussions, jumping from issue to issue in a wikipedia navigation style.

I've never experienced any issue here though, but good to have this in the background just in case.

Thanks for the review ;)